src/test/java/com/hyperwallet/clientsdk/HyperwalletIT.java

```diff
@@ -1151,6 +1158,13 @@ public void testListPaymentStatusTransitions() throws Exception {
         assertThat(returnValue.getData().get(1).getTransition(), is(equalTo(RECALLED)));
         assertThat(returnValue.getData().get(1).getFromStatus(), is(equalTo(COMPLETED)));
         assertThat(returnValue.getData().get(1).getToStatus(), is(equalTo(RECALLED)));
+        if (returnValue.getLinks() != null) {
+            HyperwalletLink actualHyperwalletLink = returnValue.getLinks().get(0);
+            HyperwalletLink expectedHyperwalletLink = returnValue.getLinks().get(0);
+            assertThat(actualHyperwalletLink.getHref(), is(equalTo(expectedHyperwalletLink.getHref())));
```

Comment:
This statement and the following two statements always return true as you are setting the value from returnValue.getLinks().get(0) to actualHyperwalletLink and expectedHyperwalletLink. Pls check the returnValue's link to that of hyperWalletLink

Reply:
addressed the review comment . thanks @Rrathinasabapath